frontend/src/components/App/Notifications/List/List.tsx

```diff
@@ -104,11 +104,31 @@ export default function NotificationList() {
           <Icon icon="mdi:dots-vertical" />
         </IconButton>
         <Menu anchorEl={anchorEl} open={Boolean(anchorEl)} onClose={handleClose}>
-          <MenuItem onClick={markAllAsRead} disabled={!hasUnseenNotifications}>
-            <Typography color={'primary'}>{t('translation|Mark all as read')}</Typography>
```

Comment:
setting color as `<Typography color="text.primary">...` fixes this
there's no need for any additional styles 

```tsx
 <MenuItem onClick={markAllAsRead} disabled={!hasUnseenNotifications}>
    <Typography color="text.primary">{t('translation|Mark all as read')}</Typography>
  </MenuItem>
```

<img width="189" height="177" alt="Image" src="https://github.com/user-attachments/assets/a704d6da-0a22-4dea-a2df-77956d8d63ed" />

<img width="193" height="191" alt="Image" src="https://github.com/user-attachments/assets/65e0e77c-051b-40ad-8129-92ab3ed8bab2" />